integrations/yocto.rst

```diff
@@ -0,0 +1,296 @@
+
+   .. image:: /images/yocto/conan-yocto_cross-build.png
+       :height: 600 px
+       :width: 600 px
+       :align: center
+
+3. Once the cross-built packages are available in Artifactory, the application can be directly deployed to the Yocto image without building
+   it from sources again.
+
+   .. image:: /images/yocto/conan-yocto_deploy.png
+       :height: 450 px
+       :width: 450 px
+       :align: center
+
+Creating Conan packages with Yocto's SDK
+========================================
+
+Prepare your recipes
+--------------------
+
+First of all, the recipe of the application to be deployed to the final image should have a
+:ref:`deploy() method <reference_conanfile_methods_deploy>`. There you can specify the files of the application
+needed in the image as well as any other from its dependencies (like shared libraries or assets):
+
+.. code-block:: shell
+
+   conan install
+   :caption: *conanfile.py*
+   :emphasize-lines: 28-31
+
+    from conan import ConanFile
+    from conan.tools.files import copy
+    import os
+
+
+    class FoobarConan(ConanFile):
+        name = "foobar"
+        ...
+
+    def package(self):
+        copy(self, "*.h", dst=os.path.join(self.package_folder, "include"), src="hello")
+        copy(self, "*.so", dst=os.path.join(self.package_folder, "lib"), keep_path=False)
+        copy(self, "*.a", dst=os.path.join(self.package_folder, "lib"), keep_path=False)
+        copy(self, "foobar", dst=os.path.join(self.package_folder, "bin"), keep_path=False)
+
+    def deploy(self):
+        # Deploy everything from this eclipse/mosquitto package
+        copy(self, "*", src=self.package_folder, dst=self.deploy_folder)
+
+Another option is using the :ref:`deployers<reference_extensions_deployers>`,
+ which will copy all artifacts, including package dependencies to your installation folder.
+
+
+Setting up a Yocto SDK
+----------------------
+
+Yocto SDKs are completely self-contained, there is no dependency on libraries of the build machine or tools installed in it. The SDK is a
+cross-building toolchain matching the target and it is generated from that specific configuration. This means that you will have to use a
+different SDK toolchain to build for a different target architecture or that some SDK's may have specific settings to enable some system
+dependency of the final target and those libraries will be available in the SDK.
+
+You can `create your own Yocto SDKs <https://docs.yoctoproject.org/sdk-manual/appendix-obtain.html#building-an-sdk-installer>`_
+or download and use `the prebuilt ones <http://downloads.yoctoproject.org/releases/yocto/yocto-5.1.2/toolchain/x86_64/>`_.
+
+**In the case that you are using CMake** to create the Conan packages, Yocto injects a toolchain that configures CMake to only search for
+libraries in the rootpath of the SDK with
+`CMAKE_FIND_ROOT_PATH <https://cmake.org/cmake/help/v3.31/variable/CMAKE_FIND_ROOT_PATH.html>`_. This is
```

Comment:
No need to link CMAKE_FIND_ROOT_PATH html page

Reply:
Done by the commit 53edd5945d8c26f1b11f67cdc7efc4769d5b5d91